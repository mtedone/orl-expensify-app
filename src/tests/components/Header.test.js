import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import React from 'react';
import Header from '../../components/Header';

test('should render Header correctly', () => {
  const wrapper = shallow(<Header />);
  expect(wrapper.debug()).toMatchSnapshot();
});
