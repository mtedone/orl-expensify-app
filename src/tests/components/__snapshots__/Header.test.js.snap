// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
"<header>
  <h1>
    Expensify
  </h1>
  <NavLink to=\\"/\\" activeClassName=\\"is-active\\" exact={true}>
    Dashboard
  </NavLink>
  <NavLink to=\\"/create\\" activeClassName=\\"is-active\\">
    Create Page
  </NavLink>
  <NavLink to=\\"/help\\" activeClassName=\\"is-active\\">
    Help
  </NavLink>
</header>"
`;
